Sample-App/app/src/main/java/com/adobe/marketing/mobile/sampleapp/EdgeIdentityTab.java

```diff
@@ -105,6 +120,58 @@ public void call(String ecid) {
                 });
             }
         });
+
+        // Default hint for how to enable ad ID features; overwritten by actual implementation when ad ID features are enabled.
+        Button btnUpdateAdId = getView().findViewById(R.id.btn_edge_identity_get_gaid);
+        btnUpdateAdId.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(final View v) {
+                Log.d(LOG_TAG,"For complete instructions on how to enable ad ID features, please see ./Documentation/README.md#advertising-identifier");
```

Comment:
Maybe something for another PR, but it would be nice to have an in app message with this warning, such as a [toast](https://developer.android.com/guide/topics/ui/notifiers/toasts) or a [snackbar](https://material.io/components/snackbars).

Reply:
@kevinlind what do you think about updating the text that would normally show the GAID value to show the text?

Reply:
Yes, that is an option too. I didn't think of that.